lib/index.js

```diff
@@ -28,39 +30,79 @@ function S3(uri, callback) {
     // If a uri of the form s3://[bucket]/[object] is passed,
     // generate implied HTTP resource and pass upstream for
     // retrieval to node-tilejson.
```

Comment:
Yeah, good catch. Adjusted this, as it is still kind of confusing how we bounce from `s3://` to `https://` and back again